src/main/java/io/castle/client/api/CastleApi.java

```diff
@@ -323,7 +323,7 @@
      * Makes a sync POST request to the impersonate endpoint.
      *
      * @param userId user unique ID
-     * @param impersonator user unique ID
+     * @param impersonator description of impersonator eg email
```

Comment:
Description of the impersonator, e.g. email (UK)
or
Description of the impersonator, e.g., email (US)
if I'm not mistaken